### PR TITLE
Add "Populate Help Text on the Payment Processor Administrator Screen" section

### PR DESCRIPTION
**Pull request by alifrumin**
Add a section on how to Populate Help Text on the Payment Processor Administrator Screen for your new processor.

### CODE REVIEW

File: docs/extensions/payment-processors/create.md [hunk 1]
```diff
@@ -613,6 +613,28 @@ else {
 
 CRM_Utils_System::redirect( $finalURL );
 ```
+### Populate Help Text on the Payment Processor Administrator Screen
+To populate the blue help icons for the settings fields needed for your payment processor on the  "Settings - Payment Processor" form (`CiviCRM Admin bar -> Adminster -> System Settings -> Payment Processors`) follow the steps below:
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
To populate the blue help icons for the settings fields needed for your payment processor at **Administer -> System Settings -> Payment Processors** follow the steps below:
```

Style guide fixes.

File: docs/extensions/payment-processors/create.md [hunk 2]
```diff
@@ -613,6 +613,28 @@ else {
...
+
+1. Add a template file to your extension with a `{htxt id='$ppTypeName-live-$fieldname'}` section for each settings field you are using for example, help text for the user-name filed for a payment processor with the name 'AuthNet' would look like:
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
1. Add a template file to your extension with a `!#twig {htxt id='$ppTypeName-live-$fieldname'}` section for each settings field you are using.

    **Example:**

    The help text for the `user-name` field for a payment processor with the name 'AuthNet' would be implemented with code like this:
```

File: docs/extensions/payment-processors/create.md [hunk 3]
```diff
@@ -613,6 +613,28 @@ else {
...
+
+```
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
    ```twig
```

File: docs/extensions/payment-processors/create.md [hunk 4]
```diff
@@ -613,6 +613,28 @@ else {
...
+{htxt id='AuthNet-live-user-name'}
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
    {htxt id='AuthNet-live-user-name'}
```

File: docs/extensions/payment-processors/create.md [hunk 5]
```diff
@@ -613,6 +613,28 @@ else {
...
+{ts}Generate your API Login and Transaction Key by logging in to your Merchant Account and navigating to <strong>Settings &raquo; General Security Settings</strong>.{/ts}</p>
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
    {ts}Generate your API Login and Transaction Key by logging in to your Merchant Account and navigating to <strong>Settings &raquo; General Security Settings</strong>.{/ts}</p>
```

File: docs/extensions/payment-processors/create.md [hunk 6]
```diff
@@ -613,6 +613,28 @@ else {
...
+{/htxt}
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
    {/htxt}
```

File: docs/extensions/payment-processors/create.md [hunk 7]
```diff
@@ -613,6 +613,28 @@ else {
...
+```
+
+see [core /templates/CRM/Admin/Page/PaymentProcessor.hlp](https://github.com/civicrm/civicrm-core/blob/master/templates/CRM/Admin/Page/PaymentProcessor.hlp) for examples.
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
    see [core /templates/CRM/Admin/Page/PaymentProcessor.hlp](https://github.com/civicrm/civicrm-core/blob/master/templates/CRM/Admin/Page/PaymentProcessor.hlp) for further examples.
```

File: docs/extensions/payment-processors/create.md [hunk 8]
```diff
@@ -613,6 +613,28 @@ else {
...
+2. Add that template to the `CRM_Admin_Form_PaymentProcessor` form using a buildForm hook like so:
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
1. Add that template to the `CRM_Admin_Form_PaymentProcessor` form using a buildForm hook like so:
```

File: docs/extensions/payment-processors/create.md [hunk 9]
```diff
@@ -613,6 +613,28 @@ else {
...
+
+```php
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
    ```php
```

File: docs/extensions/payment-processors/create.md [hunk 10]
```diff
@@ -613,6 +613,28 @@ else {
...
+ if ($formName == 'CRM_Admin_Form_PaymentProcessor') {
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
    if ($formName == 'CRM_Admin_Form_PaymentProcessor') {
```

File: docs/extensions/payment-processors/create.md [hunk 11]
```diff
@@ -613,6 +613,28 @@ else {
...
+    $templatePath = realpath(dirname(__FILE__) . "/templates");
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
        $templatePath = realpath(dirname(__FILE__) . "/templates");
```

File: docs/extensions/payment-processors/create.md [hunk 12]
```diff
@@ -613,6 +613,28 @@ else {
...
+    CRM_Core_Region::instance('form-buttons')->add(array(
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
        CRM_Core_Region::instance('form-buttons')->add(array(
```

File: docs/extensions/payment-processors/create.md [hunk 13]
```diff
@@ -613,6 +613,28 @@ else {
...
+      'template' => "{$templatePath}/{TEMPLATE FILE NAME}.tpl",
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
          'template' => "{$templatePath}/{TEMPLATE FILE NAME}.tpl",
```

File: docs/extensions/payment-processors/create.md [hunk 14]
```diff
@@ -613,6 +613,28 @@ else {
...
+    ));
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
        ));
```

File: docs/extensions/payment-processors/create.md [hunk 15]
```diff
@@ -613,6 +613,28 @@ else {
...
+  }
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
      }
```

File: docs/extensions/payment-processors/create.md [hunk 16]
```diff
@@ -613,6 +613,28 @@ else {
...
+ ```
```
Comment (MikeyMJCO) [inline docs/extensions/payment-processors/create.md]:
```suggestion
     ```
```

**Comment by alifrumin** [comment]
@MikeyMJCO thank you for this review!!!

I approved all of your suggested changes and will work to be more attentive to the style guide going forward when making updates to documentation. Let me know if you want me to squash these commits.

**Pull request merged**